settings/provisioner/foreman/foreman.yml

```diff
@@ -0,0 +1,3 @@
+---
+provisioner:
```

Comment:
Can't this be added automatically with the ir-provision? why do I need the extra file?